docs/intentReaderHandlers/RunTestHandler.md

```diff
@@ -0,0 +1,338 @@
+4. Formats the result and sends the response back to IntentReader.
+
+## Usage
+* Request Format
+
+```json
+    {
+        "action": "search",
+        "data": {
+            "query": {
+                "task": "runTest",
+                "params": {
+                    "certification": true,
+                    "exceptionMethods": [
+                        {
+                            "method": "<methodName>",
+                            "param": {}
+                        }
+                    ],
+                    "methodsToBeExcluded": [],
+                    "modulesToBeExcluded": []
+                },
+                "action": "<CORE/MANAGE/FIREBOLT-ALL>",
+                "context": {
+                    "communicationMode": "<mode>"
+                },
+                "metadata": {},
+                "asynchronous": false,
+                "appType": "<appType>"
+            }
+        },
+        "context": {
+            "source": "device"
+        }
+    }
+```
+
+### Parameters
+
+| Key                       | Description                                                                                                                       | Required? |
+|---------------------------|-----------------------------------------------------------------------------------------------------------------------------------|-----------|
+| task                      | "runTest"- Its a static value and should not be changed for this handler.                                                         | Y         |
+| params                    | Required  params for  the intent.                                                                                                 | optional  |
+| exceptionMethods          | Inner field of params where we pass the list of methods and corresponding params to be excluded from sanity suite runs.           | optional  |
+| methodsToBeExcluded       | Inner field of params where we pass the list of methods to be excluded for test execution.                                        | optional  |
+| modulesToBeExcluded       | Inner field of params where we pass the list of modules to be excluded for test execution.                                        | optional  |
+| action                    | It indicates whether its core, manage or all suite run.                                                                           | Y         |
+| communicationMode         | It indicates whether the test is run on transport or sdk mode mode.                                                               | optional  |
+| metadata                  | The info: related to target platform like deviceModel, devicePartner.                                                             | optional  |
+| appType                   | Corresponding intent is launching on which app.                                                                                   | Y         |
```

Comment:
exceptionMethods | Inner field of params where we pass the list of methods and corresponding params to be excluded from sanity suite runs. 

This is wrong. exceptionMethods are not excluded from suite executions. Give proper information here



<!--EndFragment-->
</body>
</html>

Reply:
updated